ert_shared/storage/endpoints/ensembles.py

```diff
@@ -251,3 +251,44 @@ async def create_misfit(
         )
         for response, value in responses
     )
+
+
+@router.post("/ensembles/{id}/updates", response_model=js.Update)
+async def create_observation_transformation(
+    *,
+    db: Session = Db(),
+    id: int,
+    update: js.UpdateCreate,
+):
+    try:
+        update_obj = ds.Update(
+            algorithm=update.algorithm,
+            ensemble_reference_id=update.ensemble_reference_id,
+        )
+        db.add(update_obj)
+        db.commit()
```

Comment:
should this be flushed instead and not committed to provide the possibility of a transaction rollback in case of exception or something went wrong?